plugins/dell/meson.build

```diff
@@ -1,4 +1,10 @@
-if get_option('plugin_dell')
+tpm2tss_dell = dependency('tss2-esys', version : '>= 2.0', required: get_option('plugin_dell'))
+libsmbios_c = dependency('libsmbios_c', version : '>= 2.4.0', required: get_option('plugin_dell'))
+
+if get_option('plugin_dell').enabled() and get_option('plugin_uefi_capsule').disabled()
+    error('plugin_dell is force-enabled but plugin_uefi_capsule is disabled, these options are incompatible')
```

Comment:
there's no cute way of doing this? e.g. `required: get_option('plugin_dell') and get_option('plugin_uefi_capsule'))`

Reply:
The cute way of doing it is Meson 0.59.0 and the `.require()` method. Using `and` only works with two booleans to yield a boolean, it will involve dropping the advantage of a feature option.

Reply:
Gotcha, thanks.